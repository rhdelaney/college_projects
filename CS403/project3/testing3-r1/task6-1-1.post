(println "trying (pfs 2 11 79)")
(stream-display 10 (apply pfs '(2 11 79)))
(println)
