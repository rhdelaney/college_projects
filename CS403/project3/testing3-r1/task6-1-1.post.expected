trying (pfs 2 11 79)
(2 4 8 11 16 22 32 44 64 79 ...)
